config/config.example.yml

```diff
@@ -7,5 +7,6 @@ enable_message_streaming: true  # if set, messages will be shown to user word-by
 
 # prices
 chatgpt_price_per_1000_tokens: 0.002
+chatgpt_gpt4_price_per_1000_tokens: 0.03
```

Comment:
gpt 4 pricing is indeed 0.03$ for 1k prompting but its the double for completion. We might want to be able to take that into acount

Reply:
I didn't add the completions api for GPT-4 as I didn't have the access for testing yet.

Naming convention I was following was:
Prompting APIs: chatgpt_<model>_per_N_tokens.

Completions APIs:
<model>_per_N_tokens